tests/Kernel/LanguageSwitcherTest.php

```diff
@@ -15,6 +16,91 @@ class LanguageSwitcherTest extends MultilingualAbstractKernelTestBase {
    * Test language switcher link list rendering.
    */
   public function testLanguageSwitcherLinkListRendering(): void {
+    $this->assertDefaultLanguageBlock();
+
+    $lang_config = $this->container->get('config.factory')->get('language.negotiation');
+
+    // Create the Icelandic language.
+    $language = ConfigurableLanguage::createFromLangcode('is');
+    $language->save();
+    // Load the language object.
+    $icelandic = $this->container->get('language_manager')->getLanguage('is');
```

Comment:
You know the lang ID. It's `is`, you load it based on that. Also, why you still loading it since you have it from when you created it above at line 24.